lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/evaluator/pojos/ContextMessagePOJO.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.reef.runtime.common.driver.evaluator.pojos;
+
+import org.apache.reef.annotations.audience.DriverSide;
+import org.apache.reef.annotations.audience.Private;
+import org.apache.reef.proto.ReefServiceProtos;
+
+/**
+ *  DriverSide representation of ContextMessageProto.
+ */
+@DriverSide
+@Private
+public class ContextMessagePOJO {
+
+  private byte[] message;
+  private String sourceId;
+
+  public ContextMessagePOJO(final ReefServiceProtos.ContextStatusProto.ContextMessageProto proto){
```

Comment:
Does this constructor need to be `public`? The same question arises for all the other newly added classes / constructors. I did some checks and for instance the one for `TaskMessage` can be package-private. Please check and make the new constructors the least visible.